versions.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+########################################################################
+
+# These variables specify the version of these dependencies that we
+# want. This affects both checking out the repo as well as installation
+# of built artificats.
+
+# If tree sitter in the future has more consistant releases, or better ways 
+# of locking the version, adjust this to be versions instead of commits
+
+# TREE-SITTER REPOS
+# Python -- Merge pull request #193 from tree-sitter/tausbn/... -- Feb 28, 23
+export tree_sitter_python_hash=62827156d01c74dc1538266344e788da74536b8a
```

Comment:
Instead of using a random hash, it'd probably be better to use a hash associated with a specific tag as that's in theory less likely to be "broken" vs whatever commit off the main branch.

However, I think it's possible that they've not created a tag for the latest version (like for `tree_sitter_python`) at which point we'd want the hash for the commit that the tag should be pointed at.

Reply:
Should the hash be for the most recent tag, or a commit more recently?
Like, tree-sitter-cpp, the last tag created was in 2021, and there have been significant changes since then.
I'm not quite sure what commit tags should point to.